code/modules/reagents/chemistry/reagents/impure_reagents.dm

```diff
@@ -0,0 +1,37 @@
+//Reagents produced by metabolising/reacting fermichems inoptimally, i.e. inverse_chems or impure_chems
+//Inverse = Splitting
+//Invert = Whole conversion
+
+//Causes slight liver damage, and that's it.
+/datum/reagent/impure
```

Comment:
```suggestion
/datum/reagent/impurity
```
This is bike shedding. Feel free to resolve and ignore if you don't think it adds clarity.

Reply:
This makes searchign less cluttered, so I'll implement this